tests/components/redemptions/RedemptionCard.test.jsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import { mount } from 'enzyme';
+import RedemptionCard from '../../../src/components/redemptions/RedemptionCard';
+import activity from '../../../src/fixtures/activity';
+
+describe('<RedemptionCard />', () => {
+  const wrapper = mount.bind(
+    null,
+    <RedemptionCard {...activity} />,
+  );
+
+  it('should render without crashing', () => {
+    expect(wrapper).not.toThrowError();
+  });
+
+  it('should show user details', () => {
+    const showUserDetails = true;
+    const card = mount.bind(
+      null,
+      <RedemptionCard showUserDetails={showUserDetails} {...activity} />,
+    );
+    expect(card().find('.activity__userName').length).toBe(1);
+  });
+
+  it('should render activity details', () => {
+    expect(wrapper().find('.activity').length).toBe(1);
+  });
+});
```

Comment:
Is this component receiving props? If yes, what is the relationship between props received and `activity__userName `,  classes found?

Reply:
It is only displayed when the props are received

Reply:
cool